spec/magma_attribute_spec.rb

```diff
@@ -77,6 +70,102 @@
     end
   end
 
+  describe "#revision_to_loader" do
+    it "returns entry for editable string backed options" do
+      attribute = Magma::StringAttribute.create(
+        project_name: :project,
+        attribute_name: "name", 
+        model_name: :model, 
+        description: "Old name")
+
+      entry = attribute.revision_to_loader(:description, "New name")
+
+      expect(entry).to eq([:name, "New name"])
+    end
+
+    it "returns entry for editable JSON backed options" do
+      attribute = Magma::MatchAttribute.create(
+        project_name: :project,
+        attribute_name: "name", 
+        model_name: :model,
+        validation: { type: "Array", value: [1, 2, 3] })
+
+      # Call attribute#validation_object to verify the cached validation_object
+      # gets reset
+      attribute.validation_object
```

Comment:
We probably don't need this anymore since `#validation_object` isn't cached anymore.